### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by kgriffo**
Completed front end setup.

## UI preview:
front-end-ui-first

### CODE REVIEW

**Comment by kgriffo** [comment]
@semmypurewal I forgot to add you as a reviewer to this PR! Sorry about that.

File: .gitignore
Comment (semmypurewal) [inline .gitignore:1]: You'll want to add `node_modules` to your `.gitignore` as well. And now you'll need to remove the already added `__pycache__` directory to make sure it takes effect (adding something to your .gitignore after its already added doesn't remove it).

**Review by semmypurewal — APPROVED** [review]
Looks good! Thank you for including a screenshot!

File: .gitignore
Comment (kgriffo) [reply .gitignore:1]: Sure thing, but I'm confused as to how to do this most efficiently considering I have these directories across different branches. Should I just wait until I have my most recent code merged to main, pull, then use git rm --cached -r on main?

**Pull request merged**
